File: projectNew/old_src/RoomManager.java
package com.f14.bg.hall;

/**
 * 游戏房间管理器
 *
 * @author F14eagle
 */
public class RoomManager {
    protected static int idseq = 1000;
    // private static Logger log = Logger.getLogger(RoomManager.class);

    /**
     * 生成房间id
     *
     * @return
     */
    public synchronized static int generateRoomId() {
        return idseq++;
    }

}

File: projectNew/src/com/f14/bg/hall/RoomManager.kt
package com.f14.bg.hall

/**
 * Created by 吹风奈奈 on 2017/7/18.
 */

object RoomManager {
    private var idseq = 1000

    /**
     * 生成房间id
     * @return
     */
    @Synchronized
    fun generateRoomId() = idseq++
}
